DE-image-featurizer/evaluate/evaluator.py

```diff
@@ -200,11 +193,13 @@ def _load_featurizer(self, featurizer):
             Accepted values for names are 'hog', 'lbp', 'orb' and 'sift'.
         """
         if featurizer.lower() in FEATURIZER_DICT:
+            self.traditional = True
             self.featurizer = FEATURIZER_DICT[featurizer.lower()].fit_transform
         else:
+            self.traditional = False
             self.featurizer = hub.KerasLayer(featurizer,
-                                             input_shape=(*self.input_shape,3), trainable=False)
-        self.preprocessor = tf.keras.Sequential([tf.keras.layers.Resizing(*self.input_shape),
+                                             input_shape=(224, 224, 3), trainable=False)
+            self.preprocessor = tf.keras.Sequential([tf.keras.layers.Resizing(224, 224),
```

Comment:
@rahulthakurakridata What's the idea behind removing `self.input_shape`? Shouldn't this be configurable by the user?